packages/esm-patient-chart-app/src/visit/visits-widget/current-visit-summary.test.tsx

```diff
@@ -1,34 +1,38 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import { useVisit, getConfig } from '@openmrs/esm-framework';
-import { waitForLoadingToFinish } from 'tools';
-import CurrentVisitSummary from './current-visit-summary.component';
+import { getConfig } from '@openmrs/esm-framework';
+import { mockPatient, waitForLoadingToFinish } from 'tools';
+import { usePatientChartStore } from '@openmrs/esm-patient-common-lib';
+import CurrentVisitSummary from './current-visit-summary.extension';
 
 const mockGetConfig = jest.mocked(getConfig);
-const mockUseVisits = jest.mocked(useVisit);
+const mockUsePatientChartStore = jest.mocked(usePatientChartStore);
+
+jest.mock('@openmrs/esm-patient-common-lib/src/store/patient-chart-store', () => ({
+  usePatientChartStore: jest.fn(),
+}));
 
 describe('CurrentVisitSummary', () => {
   test('renders an empty state when there is no active visit', () => {
-    mockUseVisits.mockReturnValueOnce({
-      activeVisit: null,
-      currentVisit: null,
-      currentVisitIsRetrospective: false,
-      error: null,
-      isLoading: false,
-      isValidating: false,
-      mutate: jest.fn(),
+    mockUsePatientChartStore.mockReturnValue({
+      patientUuid: mockPatient.id,
+      patient: mockPatient,
+      visitContext: null,
+      mutateVisitContext: null,
+      setPatient: jest.fn(),
+      setVisitContext: jest.fn(),
     });
-
-    render(<CurrentVisitSummary patientUuid="some-uuid" />);
+    render(<CurrentVisitSummary patientUuid={mockPatient.id} />);
     expect(screen.getByText(/current visit/i)).toBeInTheDocument();
     expect(screen.getByText('There are no active visit to display for this patient')).toBeInTheDocument();
   });
 
```

Comment:
```suggestion

  test('returns null when patientUuid does not match store patientUuid', () => {
    mockUsePatientChartStore.mockReturnValue({
      patientUuid: 'different-patient-id',
      patient: mockPatient,
      visitContext: null,
      mutateVisitContext: null,
      setPatient: jest.fn(),
      setVisitContext: jest.fn(),
    });
    render(<CurrentVisitSummary patientUuid={mockPatient.id} />);
    expect(screen.queryByText(/current visit/i)).not.toBeInTheDocument();
  });
 
```

This might be a useful test.